source/account_structure/index.html.md.erb

```diff
@@ -1,92 +1,68 @@
 ---
-title: How accounts work
-last_reviewed_on: 2021-01-08
+title: Setting up multiple services
+last_reviewed_on: 2021-02-03
 review_in: 6 months
 weight: 40
 ---
 
-# How accounts work
+# Setting up multiple services
+  
+If you have more than one online service, you can set up your GOV.UK Pay account and payment service provider (PSP) accounts so it's easier to manage your payments and services.
 
-GOV.UK Pay does not limit how many services you can integrate with it and how many accounts you can have with us.
-GOV.UK Pay test and live accounts are free to create and maintain but your payment service provider may charge you for the underlying accounts.
+## Decide where to send users
 
-## Account structure examples
+You can send your users to make payments with either:
 
-If you do integrate multiple services, you can treat them separately or as a
-combined single service within GOV.UK Pay and your payment service provider (PSP).
-This gives 3 possible account configurations.
+- a different GOV.UK Pay service for each of your services
```

Comment:
Hopefully the way things are broken down in 81e419d means that we can guide users through their particular scenario. I'd like to start without specific scenarios if possible because I'm not sure they'll be particularly helpful - from the support tickets I'm not sure each team's situation maps nicely to others, so specific examples may lead to even more mental overhead. Happy to add later if testing/evidence shows we need them though.